extension/upload-button.js

```diff
@@ -0,0 +1,35 @@
+/* globals pageDetect, icons */
+
+const isMac = /Mac OS X/.test(window.navigator.oscpu);
```

Comment:
OK, going by this [SO answer](http://stackoverflow.com/a/11752084) it seems like `window.navigator.platform` is the best option.